examples/siro_sandbox/sandbox_app.py

```diff
@@ -800,6 +835,10 @@ def depth_to_rgb(obs):
 
         self._update_help_text()
 
+        # store post_sim_update_dict for frozen sim mode
+        # when episode is over or when task is completed
+        self._prev_post_sim_update_dict = post_sim_update_dict
```

Comment:
This feels like a hack. Why is this necessary?

Reply:
At a high level, we should get away from the idea that, because the simulation is paused, we're necessarily going to render the same thing or otherwise "re-use" the `update_dict`. That is not the case. E.g. I can imagine implementing a camera that kinda circles the agent during this time (even though the sim is paused).

Reply:
One thing that probably needs to change is this logic in `replay_gui_app_renderer.py`:
```
        if len(keyframes):
            self._need_render = True
```

When the sim is paused, we may still need to re-render. Again, the example of the camera moving.

Reply:
Oh, got you! I misunderstood the task. Fixed!